Web/src/app/app.component.ts

```diff
@@ -76,9 +75,11 @@ export class AppComponent  {
 
   getFromDb() {
     this.textObservable = this.db.list('/').valueChanges();
-    this.textObservable.subscribe(res => this.dbText.text = Object.assign(res));
-    setTimeout(() => console.log(this.dbText.text[0]), 1000);
-    setTimeout(() => this.postWatson(this.dbText.text[0]), 1000);
+    this.textObservable.subscribe(res => {
+      this.postWatson(this.dbText.text[0])
+    });
+    
+    // setTimeout(() => console.log(this.dbText.text[0]), 1000);
```

Comment:
do we still want to keep this commented out?

Reply:
we should remove that comment